roles/apache/tasks/main.yml

```diff
@@ -81,6 +81,14 @@
     notify:
       - reload apache
 
+  - name: remove TLSv1 and TLSv1.1 from ssl.conf
+    lineinfile: dest=/etc/apache2/mods-available/ssl.conf
+                regexp="^\s*SSLProtocol"
+                line='\tSSLProtocol All -SSLv2 -SSLv3'
```

Comment:
Don't we want this to use the values from `roles/apache/defaults/main.yml`?  That line is `SSLProtocol All -SSLv2 -SSLv3 -TLSv1 -TLSv1.1`

Reply:
We should probably also determine what other parts of `apache.ssl.settings` should be set in this file, since they don't appear to get used anywhere else.  One way would be to convert `apache.ssl.settings` from a list to a dict.
```yaml
  ssl:
    settings: |
      SSLEngine: on
      SSLProtocol: All -SSLv2 -SSLv3 -TLSv1 -TLSv1.1
      SSLHonorCipherOrder: On
      SSLCompression: off
      # Add six earth month HSTS header for all users...
      Header: always set Strict-Transport-Security "max-age=15768000"
      SSLCipherSuite: 'EDH+CAMELLIA:EDH+aRSA:EECDH+aRSA+AESGCM:EECDH+aRSA+SHA384:EECDH+aRSA+SHA256:EECDH:+CAMELLIA256:+AES256:+CAMELLIA128:+AES128:+SSLv3:!aNULL:!eNULL:!LOW:!3DES:!MD5:!EXP:!PSK:!DSS:!RC4:!SEED:!ECDSA:CAMELLIA256-SHA:AES256-SHA:CAMELLIA128-SHA:AES128-SHA'
```
And then tweak your `lineinfile` task to loop over them

```yaml
 - name: apply apache ssl settings
   lineinfile: dest=/etc/apache2/mods-available/ssl.conf
               regexp="^\s*{{ item.key }}"
               line='\t{{ item.key }} {{ item.value }}'
   with_dict: "{{ apache.ssl.settings }}"
```